docker-compose-test.yml

```diff
@@ -20,6 +20,19 @@ services:
     volumes:
       - test-pgdata:/var/lib/postgresql/data
 
+  gotenberg:
+    image: gotenberg/gotenberg:7
```

Comment:
looks like there's a [gotenberg 8](https://github.com/gotenberg/gotenberg/releases/tag/v8.0.0) now, could address in follow up?